apps/win32/package.json

```diff
@@ -32,6 +32,9 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.104.0",
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-native-fontawesome": "^0.3.0",
```

Comment:
Also update fluent testers package.json?

Reply:
That's probably a better place for it!